alvr/server/cpp/alvr_server/Settings.cpp

```diff
@@ -130,6 +130,10 @@ void Settings::Load()
 		m_leftControllerRotationOffset[2] = leftControllerRotationOffset[2].get<double>();
 
 		m_hapticsIntensity = config.get("haptics_intensity").get<double>();
+		m_hapticsAmplitudeCurve = config.get("haptics_amplitude_curve").get<double>();;
+		m_hapticsMinDuration = config.get("haptics_min_duration").get<double>();;
+		m_hapticsLowDurationAmplitudeMultiplier = config.get("haptics_low_duration_amplitude_multiplier").get<double>();;
+		m_hapticsLowDurationRange = config.get("haptics_low_duration_range").get<double>();;
```

Comment:
Is this `double` 32-bit like the Rust-side `f32`? Not really a big deal but still something I'm curious about.

Reply:
I don't think so
All float options get as a double but not sure why

Reply:
Good catch
(How did I miss that?)